tests/conftest.py

```diff
@@ -461,6 +490,28 @@ def table_update(req: tsi.TableUpdateReq) -> tsi.TableUpdateRes:
         )
         return client.server.table_update(req)
 
+    @app.post("/feedback/create")
```

Comment:
`POST /feedback` is the canonical create endpoint format for a RESTful API.

I know we don't respect any RESTful convention anywhere but it would be good to start.

Reply:
Ah no sorry, ignore my comment. Let's keep this disccussion for API v2.